### PR TITLE
[Chips] Auto Generated README update.

### PR DESCRIPTION
**Pull request by mohammadcazig**

### CODE REVIEW

File: components/Chips/README.md [hunk 1]
```diff
@@ -1,44 +1,57 @@
 <!--docs:
-title: "Chips"
+title: "Chip"
 layout: detail
 section: components
-excerpt: "Chips represent complex entities in small blocks, such as a contact."
+excerpt: "<#short_description#>"
```
Comment (jverkoey) [inline components/Chips/README.md]: Need to provide a description.

File: components/Chips/README.md [hunk 2]
```diff
@@ -1,44 +1,57 @@
 <!--docs:
-title: "Chips"
+title: "Chip"
```
Comment (jverkoey) [inline components/Chips/README.md]: Component name should be Chips in .vars.

**Review by jverkoey — DISMISSED** [review]
Minor changes.

File: components/Chips/README.md (hunk 2 above)
Comment (mohammadcazig) [reply components/Chips/README.md]: done.

File: components/Chips/README.md (hunk 1 above)
Comment (mohammadcazig) [reply components/Chips/README.md]: done.

**Review by jverkoey — APPROVED** [review]

**Comment by jverkoey** [comment]
Force merge once website generation passes.

**Pull request merged**